api/v1alpha1/envoyproxy_types.go

```diff
@@ -86,6 +87,20 @@ type EnvoyProxySpec struct {
 	//
 	// +optional
 	Shutdown *ShutdownConfig `json:"shutdown,omitempty"`
+
+	// TLS is the TLS configuration for the Envoy proxy to use when connecting to
+	// backend services and external services, such as ExtAuth, ALS, OpenTelemetry, etc.
+	// +optional
+	TLS *EnvoyTLSConfig `json:"tls,omitempty"`
```

Comment:
not good at TLS, is it possible need a slice here?

Reply:
It's the client cert for Envoy to use when connecting to backend services. One would be enough unless we have use cases where we want to differentiate envoy identities when connecting to different backends.🤔